docs/IR.md

```diff
@@ -55,7 +55,7 @@ The IR specification uses simple monotonically increasing numbers for its versio
   IR_VERSION = 0x0000000000000003;
 ```
 
-Operator sets use a simple version number. Each operator set version represents the combination of the most recent version of each operator.
+Operator sets use a simple version number. Each operator set version represents the combination of the most recent version of each operator at a particular point in time.
```

Comment:
**each** operator set version representing the combination of the **most recent version** of each operator was confusing to me. hopefully this matches the intended meaning.

Reply:
I'm not a good English editor/writer, so am not sure whether this editing change improves the readability and clarifies more on semantics of operator set versioning. @prasanthpul  @gramalingam @houseroad @ebarsoum 

The operator set version may be considered as a time point (normally one per release). Each operator may have its specific semantics since one time point (one operator set version). That's why operator does not have "version" itself, but "sinceversion".

Reply:
I think "Each operator set version represents a combination of a specific version of some set of operators." would be simpler. I think adding "The operator set version may considered as a time point. Each version of an operator belongs to some contiguous set of operator set versions. Normally, there is one operator set version per release of ONNX (if and only if there is a change in the set of operators or the specification of any existing operator)." would also be helpful.

Reply:
updated and added an example to versioning.md.